mesonbuild/ast/interpreter.py

```diff
@@ -243,10 +244,10 @@ def __init__(self, source_root: str, subdir: str, subproject: SubProject, subpro
                            'debug': self.func_do_nothing,
                            })
 
-    def _unholder_args(self, args: _T, kwargs: _V) -> T.Tuple[_T, _V]:
+    def _unholder_args(self, args: T.Any, kwargs: T.Any) -> T.Tuple[T.Any, T.Any]:
         return args, kwargs
 
-    def _holderify(self, res: _T) -> _T:
+    def _holderify(self, res: T.Any) -> T.Any:
```

Comment:
Looks suspicious here too.

Reply:
See my comment about the _unholder_args function.